manifests/params.pp

```diff
@@ -57,6 +57,8 @@
         $npm_path                  = '/usr/bin/npm'
         $repo_class                = '::nodejs::repo::nodesource'
       }
+
+      $package_provider          = 'apt'
```

Comment:
I'd prefer to set this and others (yum, zypper etc) to undef unless it's non-standard (macports? chocolatey?).

Reply:
Changed it over